src/main.cc

```diff
@@ -302,16 +301,24 @@ main(int argc, char** argv) {
        "view.filter = leeching,((and,((d.state)),((not,((d.complete))))))\n"
        "view.filter_on = leeching,event.download.resumed,event.download.paused,event.download.finished\n"
 
-       "schedule2 = view.main,10,10,((view.sort,main,20))\n"
-       "schedule2 = view.name,10,10,((view.sort,name,20))\n"
-
        "schedule2 = session_save,1200,1200,((session.save))\n"
        "schedule2 = low_diskspace,5,60,((close_low_diskspace,500M))\n"
        "schedule2 = prune_file_status,3600,86400,((system.file_status_cache.prune))\n"
 
        "protocol.encryption.set=allow_incoming,prefer_plaintext,enable_retry\n"
     );
 
+    // Only enable view sorting when not in daemon mode
+    if (!rpc::call_command_value("system.daemon")) {
+      rpc::parse_command_multiple
+        (rpc::make_target(),
+         "view.sort_new     = name,((less,((d.name))))\n"
+         "view.sort_current = name,((less,((d.name))))\n"
+         "schedule2 = view.main,10,10,((view.sort,main,20))\n"
+         "schedule2 = view.name,10,10,((view.sort,name,20))\n"
+        );
+    }
```

Comment:
Daemon mode still uses views, so the name view should be sorted.

However this could be changed to be added on `event.system.startup_done`, which likely should also be used to enable scgi if opened.